src/main/java/com/offdk/play/web/SlackCommandParamHandler.java

```diff
@@ -27,11 +32,33 @@ public Object resolveArgument(MethodParameter param, ModelAndViewContainer conta
         command=/weather
         text=94070
         response_url=https://hooks.slack.com/commands/1234/5678
-     */
-    return new SlackCommand();
-  }
+		 */
+
+		HashMap<String, String> parserMap = new HashMap<String, String>();
+
+		String[] key_value_array = request.getContextPath().split("&");
```

Comment:
Java doesn't use underscores in variables names (unless it's a class variable, and even then, it's obsolete since java has the keywork this). Instead, all variable names should be camelCased, with the first letter being lowercase.